src/coreclr/vm/dllimport.cpp

```diff
@@ -2511,6 +2493,31 @@ class DispatchStubState : public StubState // For CLR-to-COM late-bound/eventing
 
 #endif // FEATURE_COMINTEROP
 
+namespace
+{
+    CorInfoCallConvExtension GetCallConvValueForPInvokeCallConv(CorPinvokeMap callConv)
+    {
+        LIMITED_METHOD_CONTRACT;
+
+        switch (callConv)
+        {
+        case 0:
+        case pmCallConvWinapi:
```

Comment:
This seems odd. WinAPI is stdcall on Windows. What is this function for?

Reply:
I'll add some comments here. This function converts a CorPinvokeMap calling convention to a CorInfoCallConvExtension calling convention, but we need to know when the user either didn't provide a calling convention or provided WinApi as compared to when the user explicitly provided the platform default calling convention (such as having `CallingConvention = CallingConvention.Stdcall` and running on Windows).

Reply:
Sounds good. Comments are warranted and should be sufficient.